packages/eslint-config-react-app/index.js

```diff
@@ -315,5 +323,30 @@ module.exports = {
     'flowtype/define-flow-type': 'warn',
     'flowtype/require-valid-file-annotation': 'warn',
     'flowtype/use-flow-type': 'warn',
+
+    // https://github.com/testing-library/eslint-plugin-jest-dom
+    'jest-dom/prefer-checked': 'error',
+    'jest-dom/prefer-empty': 'error',
+    'jest-dom/prefer-enabled-disabled': 'error',
+    'jest-dom/prefer-focus': 'error',
+    'jest-dom/prefer-required': 'error',
+    'jest-dom/prefer-to-have-attribute': 'error',
+    'jest-dom/prefer-to-have-text-content': 'error',
+
+    // https://github.com/testing-library/eslint-plugin-testing-library
+    'testing-library/await-async-query': 'error',
+    'testing-library/await-async-utils': 'error',
+    'testing-library/await-fire-event': 'off',
+    'testing-library/consistent-data-testid': 'off',
+    'testing-library/no-await-sync-query': 'error',
+    'testing-library/no-debug': 'error',
+    'testing-library/no-dom-import': ['error', 'react'],
+    'testing-library/no-manual-cleanup': 'error',
+    'testing-library/no-wait-for-empty-callback': 'error',
+    'testing-library/prefer-explicit-assert': 'warn',
+    'testing-library/prefer-find-by': 'error',
+    'testing-library/prefer-presence-queries': 'error',
+    'testing-library/prefer-screen-queries': 'error',
+    'testing-library/prefer-wait-for': 'error',
```

Comment:
Let's also use the recommended React config here so it's easier to compare the differences

Reply:
Same